packages/kbn-apm-synthtrace/src/lib/apm/processors/get_breakdown_metrics.ts

```diff
@@ -27,6 +27,7 @@ const metricsetPicker = createPicker([
   'transaction.name',
   'span.type',
   'span.subtype',
+  'service.name',
```

Comment:
I think `instanceFields` should take care of this. Is there a bug somewhere else?

Reply:
Instance fields are no used in the [hash key](https://github.com/elastic/kibana/blob/e937f118bd470a66328965227a10293739168361/packages/kbn-apm-synthtrace/src/lib/apm/processors/get_breakdown_metrics.ts#L109) which is used to decide whether to create a new set. Do you think that it should be added there?

Reply:
I think so, because e.g. service.node.name should also be part of the key.

Reply:
makes sense, changed it in https://github.com/elastic/kibana/pull/140256/commits/9497df7644549bc2ef4c6e4a49218e7a046f12a6